homeassistant/components/opengarage/cover.py

```diff
@@ -79,8 +79,9 @@ class OpenGarageCover(CoverDevice):
 
     def __init__(self, args):
         """Initialize the cover."""
-        self.opengarage_url = "{}://{}:{}".format(
-            "https" if args[CONF_SSL] else "http", args[CONF_HOST], args[CONF_PORT]
+        self.opengarage_url = (
+            f"{'https' if args[CONF_SSL] else 'http'}://"
```

Comment:
```suggestion
            f"http{'s' if args[CONF_SSL]}://"
```

Then maybe you can make this f-string one line.

Reply:
Great suggestion to make this a one-liner, but I think inline if statements need to include an `else`. However, I could create a local variable beforehand and use it in the f-string, what do you think?

Do not worry, I am slowly acquiring that useful habit and I have already cleaned my repo quite a bit 😉

Reply:
Oh yeah sure, I should sleep now 😅

Or a else with empty string, as you want

Reply:
Oh I am sorry, good night then ✌

```suggestion
            f"http{'s' if args[CONF_SSL] else 's'}://"
```